src/Sarif/Writers/ConsoleLogger.cs

```diff
@@ -2,17 +2,18 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.Collections.Generic;
 using System.Globalization;
 using System.Linq;
 using System.Text;
 
 namespace Microsoft.CodeAnalysis.Sarif.Writers
 {
-    public class ConsoleLogger : IAnalysisLogger
+    public class ConsoleLogger : BaseLogger, IAnalysisLogger
     {
-        public ConsoleLogger(bool verbose, string toolName)
+        public ConsoleLogger(bool quietConsole, string toolName, IEnumerable<FailureLevel> level = null, IEnumerable<ResultKind> kind = null) : base(level, kind)
```

Comment:
>level [](start = 91, length = 5)

rename to levels and kinds, just like the other places

Reply:
Done.

---
In reply to: [561427077](https://github.com/microsoft/sarif-sdk/pull/2241#discussion_r561427077) [](ancestors = 561427077)